src/Virtual_account.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Razorpay\Api;
+
+class Virtual_account extends Entity
```

Comment:
This file can be removed now

Reply:
It was renamed through an IDE, I thought it was handled. will remove it